fastlane/Fastfile

```diff
@@ -269,41 +269,67 @@ platform :ios do
   #####################################################################################
-  desc 'Trigger the final release build on CI'
-  lane :finalize_release do |options|
-    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+  desc 'Check translation progress for all GlotPress projects'
+  lane :check_all_translations do |options|
+    abort_on_violations = false
+    skip_confirm = options.fetch(:interactive, false) == false
 
-    ios_finalize_prechecks(options)
-    
     UI.message('Checking app strings translation status...')
     check_translation_progress(
       glotpress_url: 'https://translate.wordpress.org/projects/apps/ios/dev/',
-      abort_on_violations: false
+      abort_on_violations: abort_on_violations,
+      skip_confirm: skip_confirm
     )
 
-    UI.message("Checking WordPress release notes strings translation status...")
+    UI.message('Checking WordPress release notes strings translation status...')
     check_translation_progress(
       glotpress_url: 'https://translate.wordpress.org/projects/apps/ios/release-notes/',
-      abort_on_violations: false
+      abort_on_violations: abort_on_violations,
+      skip_confirm: skip_confirm
     )
 
-    UI.message("Checking Jetpack release notes strings translation status...")
+    UI.message('Checking Jetpack release notes strings translation status...')
     check_translation_progress(
       glotpress_url: 'https://translate.wordpress.com/projects/jetpack/apps/ios/release-notes/',
-      abort_on_violations: false
+      abort_on_violations: abort_on_violations,
+      skip_confirm: skip_confirm
     )
+  end
+
+  #####################################################################################
+  # finalize_release
+  # -----------------------------------------------------------------------------------
+  # This lane finalize a release: updates store metadata, bump final version number,
+  # remove branch protection and close milestone, then trigger the final release on CI
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane finalize_release [skip_confirm:<skip confirm>] [version:<version>]
+  #
+  # Example:
+  # bundle exec fastlane finalize_release
+  # bundle exec fastlane finalize_release skip_confirm:true
+  #####################################################################################
+  desc 'Trigger the final release build on CI'
+  lane :finalize_release do |options|
+    UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+
+    ios_finalize_prechecks(options)
+
+    check_all_translations(interactive: true)
 
     download_localized_strings_and_metadata(options)
     # FIXME: (2021.06.17) This is disabled because we currently have a >256 chars string which GlotPress truncates when exporting  the `.strings` files,
```

Comment:
📝 Unrelated to this, but we should really address that FIXME at some point, in order to re-enable translation linting, especially since this is an easy fix (just split the `NSLocalizedString` that is too long in the codebase into 2 shorter ones). That would allow us to go back to ensure security at runtime (avoiding risk of crashes due to incorrect placeholders in translations)